src/assets/styles/globals.css

```diff
@@ -11,6 +11,17 @@ body {
   box-sizing: border-box;
 }
 
+.feedback-button {
```

Comment:
we can actually nix this, I previously got feedback from the design system team that this was a one-off style that shouldn't have been implemented in CB in the first place.

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15511#issuecomment-1756295839

Reply:
gotcha, removed.  Thanks!